plugins/chat/app/services/chat_message_destroyer.rb

```diff
@@ -11,6 +11,15 @@ def destroy_in_batches(chat_messages_query, batch_size: 200)
       end
   end
 
+  def trash_message(message, actor)
+    ChatMessage.transaction do
+      message.trash!(actor)
+      ChatMention.where(chat_message: message).destroy_all
+
+      ChatPublisher.publish_delete!(message.chat_channel, message)
```

Comment:
Just add FIXME, for the frontend problematic, I think we can ignore it for now, but it's good to have a trace here.